tests/resolvers.test.js

```diff
@@ -46,7 +48,51 @@ describe("User resolver", () => {
     expect(res.body.singleResult.errors).toBeUndefined();
     expect(res.body.singleResult.data).toBeDefined();
     expect(res.body.singleResult.data.registerUser).toEqual({
-      name: mockInputData.name,
+      name: mockUserInputData.name,
     });
   });
+
+  it("should throw validation error on password not matching requirements", async () => {
+    const contextValue = {
+      db: context.db,
+    };
+
+    const mockUserInputIncorrectData = {
+      email: "testAnother@email.com",
+      name: "Johny",
+      password: "password",
+    };
+
+    const res = await server.executeOperation(
+      {
+        query:
+          "mutation Mutation($input: RegisterUserInput!) {registerUser(input: $input) {name}}",
+        variables: {
+          input: mockUserInputIncorrectData,
+        },
+      },
+      {
+        contextValue,
+      }
+    );
+
+    expect(res.body.singleResult.data).toBeNull();
+    expect(res.body.singleResult.errors).toBeDefined();
+    expect(res.body.singleResult.errors).toEqual(
+      expect.arrayContaining([
```

Comment:
Is it possible to move it to a variable and avoid deep nesting?

Reply:
Nested objects moved into seperate variables.

@IvanLiaskovski